integreat_cms/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "2022.8.3"
+__version__ = "2022.8.2"
```

Comment:
Could you remove commit dd806643b1f60e8f230a93ec21e7daf331559da5 from the git history of your branch? This version bump commit was probably reverted accidentally?